db/db_impl/db_impl.cc

```diff
@@ -3246,8 +3246,8 @@ Status DBImpl::MultiGetImpl(
   return s;
 }
 
-void DBImpl::MultiGetEntity(const ReadOptions& _read_options, size_t num_keys,
-                            ColumnFamilyHandle** column_families,
+void DBImpl::MultiGetEntity(const ReadOptions& _read_options,
```

Comment:
Just swapped the location with line 3274 to align with the order we have in db.h. (MultiGetEntity for single CF and MultiGetEntity for multiple CFs)